### DISCUSSION TITLE
Modelo Dominio

### DISCUSSION THREAD

**Discussion by VeronikaEspa** [VeronikaEspa/23-24-IdSw1-SDR#3] [Show and tell]
El proceso de desarrollo del **_Modelo de Dominio_** se está gestionando de la siguiente forma:

Los miembros del equipo se han organizado en subgrupos específicos para llevar a cabo la elaboración y revisión de este Modelo de Dominio. Se realizarán iteraciones sucesivas hasta la finalización completa del modelo. Los subgrupos asignados para cada iteración son:

Primera iteración: Eduardo Lostal y Juan José Cobo.
Segunda iteración: Lucía Zamanillo y Jorge Quiñonez.
Tercera iteración: Veronika Esparragoza, Javier Cáder y José Recinos.
Este ciclo iterativo continuará hasta que se complete el desarrollo del Modelo de Dominio.

**Comment by recinosjer** [comment]
Tercera Iteración - 26/10/2023
-Javier Cáder, Veronika Esparragoza, José Recinos
-Arreglos del modelo del dominio y arreglo general del repositorio en base a lo que mencionó _Manuel Masías_ .
-Integrantes de la siguiente iteración: @EduLostal , @juanjoc711 
-Fecha tentativa de la siguiente iteración : 29/10/2023

**Comment by claraarnaiz** [comment]
Hola chicos, ya tengo creada la cuenta. 
Un saludo.

**Comment by EduLostal** [comment]
Cuarta Iteración - 30/10/2023
-Eduardo Lostal, Juan José Cobo
-Arreglos del modelo del dominio y eliminación de directorios y archivos innecesarios.
-Integrantes de la siguiente iteración: @Jorge85015  , @luciaZama 
-Fecha tentativa de la siguiente iteración : 2/10/2023

**Comment by recinosjer** [comment]
Sexta Iteración - 06/11/2023
-José Recinos, Veronika Esparragoza
-Arreglos del modelo del dominio, creación y eliminación de directorios, además de la primera iteración del diagrama estado.
-Integrantes de la siguiente iteración: @juanjoc711  , @EduLostal 
-Fecha tentativa de la siguiente iteración : 8/11/2023

**Comment by claraarnaiz** [comment]
Hola chicos! si necesitáis cualquier cosa me decís :)

**Reply by recinosjer** [reply]
Hola Clara! Te parece si: el fichaje lo dividimos en 2 ( FichajeIndividual,  FichajeDía).  El fichaje individual es cada fichaje que cada profesor hace y el fichaje del dia nos ayudará para verificar si el usuario se deslogueó. Este nos ayudará a comprender más el tema del fichaje. Gracias a esto manejaríamos posibles casos en los que no se pudo desfichar (desloguear) el usuario).

**Comment by EduLostal** [comment]
Séptima Iteración - 08/11/2023
-Eduardo Lostal, Juan José Cobo
-Arreglos del modelo del dominio (tanto en los de formato "SVG" como los "puml"...)
-Modificación del diagrama estado.
-Creación del diagrama de estados.

-Integrantes de la siguiente iteración: @Jorge85015 , @luciaZama
-Fecha tentativa de la siguiente iteración : 11/11/2023

**Comment by claraarnaiz** [comment]
Hola, perfecto. Me parece bien!
